components/demo/demo-snippet.js

```diff
@@ -68,6 +74,23 @@ class DemoSnippet extends LitElement {
 				position: sticky;
 				top: 0;
 			}
+			d2l-dropdown.settings-dropdown {
+				background-color: white;
+				border-radius: 6px;
+				box-shadow: 0 0 0 1px var(--d2l-color-celestine-minus-1);
+				position: fixed;
+				right: 1rem;
```

Comment:
I suspect we might want `left: 1rem; right: auto;` when in RTL mode. For many of our core demos, the element of interest would be in the proximity of the opener in RTL mode, and in such case having it flipped might be desirable. Having said, any demo where we are toggling to fullscreen mode probably has lots more elements of interest, and there would always be a chance of those elements being in the same proximity as the dropdown opener.

Note that the opener positioning doesn't work too well with the toolbar of the list (see list demo page). It appears to be a stacking issue... might be able to just tweak your `z-index` to resolve it.
![image](https://user-images.githubusercontent.com/9042472/202554891-fcc1f062-0ff4-4ca9-9f68-e1047b65df1a.png)

Reply:
Yeah I'm open to whatever positioning makes most sense. I'll look at the stacking issue.

Reply:
And for more context, the current behavior was working mostly fine but was not giving us an accurate representation of when our breakpoints triggered.

Reply:
Sounds good. If we find the position in RTL is problematic more often then not, we can add the RTL rule later.